frontend/src/components/MyNavbar/MyNavbar.js

```diff
@@ -0,0 +1,84 @@
+import React, {useState} from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Im not sure if we need to import the bootstrap min again. Once it is imported into index.js, it will apply to all of the components we create.

Reply:
Thank you for letting me know, I removed it.